### DISCUSSION TITLE
Hardware requirements

### DISCUSSION THREAD

**Discussion by jao-tm** [srsran/srsRAN_Project#50] [General Help] [answered]
Hi all, 
What hardware did you use when testing this software? I'm mainly interested in the computational part (server, laptop e.g.) and which throughput results you achieved with that setup.
Also, has anyone tested in a virtualized environment, for instance by running this in a VM?

**Comment by andrepuschmann** [comment, marked as answer]
Hey, the hardware requirements depend on a variety of factors - including cell bandwidth, data rates, channel conditions, etc. But to give you just one example: with a recent but standard laptop, e.g. a i7-1260P you can run a 50 MHz TDD SISO cell and achieve 112Mbit/s in the downlink and 50Mbit/s uplink simultaneously. This is with a USRP b200 mini connected over USB3.

```
           -------------DL----------------|------------------UL--------------------
 pci rnti  cqi  mcs  brate   ok  nok  (%) | pusch  mcs  brate   ok  nok  (%)    bsr
   1 4601   15   26   104M 1039  161  13% |  17.7   25    49M  594    6   1%   300k
   1 4601   15   26   111M 1047  153  12% |  17.7   25    48M  597    3   0%   300k
   1 4601   15   27   109M  998  202  16% |  17.8   25    49M  598    2   0%   300k
```

**Reply by Moussa-Guemdani** [reply]
Hello, I would like to know if 12 CPUs (i7-8700 CPU @ 3.20GHz) are enough for runing gNB (srsRAN_Project) n77 band, 20 MHz channel TDD, using USRP X310?

**Reply by andrepuschmann** [reply]
Should be enough for 20 MHz. SISO most likely, maybe even 2x2.

**Reply by mdhinwa** [reply]
Hi, I am planning to test srsRAN gNB and srsUE in a basic SISO(1×1) configuration (Single UE registration, deregistration, uplink and downlink data transfer). n78 band, 20 MHz. Would like to know if the following setup will work
<img width="488" height="114" alt="image" src="https://github.com/user-attachments/assets/45c18f57-6885-4609-a0de-0dff8d9a7078" />
I just want to confirm that b200 mini USRPs will work in this setup.

**Reply by Zplusless** [reply]
@mdhinwa  Hi，similar configuration works fine for me. But I use USRP B210
